forem/forem
# Implement Forem Organization Unified Embed

## What type of PR is this? (check all applicable)


## Description
This PR implements the [unified embed tag](https://github.com/forem/forem/issues/15099) for Forem Organization embeds.

## Related Tickets & Documents
closes https://github.com/forem/forem/issues/15567

## QA Instructions, Screenshots, Recordings
- As a user, create an organization if there isn't one. Next, create a Forem Organization liquid tag using this format: `{% embed <forem-org-url> %}`. NOTE: this is the organization's Forem-specific url (`forem-domain/org-slug`), NOT the organization's actual website URL. The Liquid Tag should render properly.
- Also create a Forem Org liquid tag using the old method: `{% organization <forem-org-slug> %}` or `{% org <forem-org-slug> %}`. This should also work, as I am leaving the old implementation in place for now.

### UI accessibility concerns?
None

## Added/updated tests?


## [Forem core team only] How will this change be communicated?
Considering the project as a whole, I shall update the Liquid Tag documentation at the end. Since I am not changing the current implementation, the present documentation is still valid. Plus, I will avoid updating the docs piecemeal, and risk confusing users.

## [optional] Are there any post deployment tasks we need to perform?
None
